node.gyp

```diff
@@ -870,7 +870,7 @@
             'deps/v8/include'
           ],
           'conditions' : [
-            ['node_use_v8_platform=="true" and node_engine=="v8"', {
+            ['node_use_v8_platform=="true"', {
```

Comment:
I remember we had an issue where we were using v8_platform incorrectly -- I think @digitalinfinity looked into it? Wasn't this the gtest issue?

Reply:
Oh yeah, nice catch! @kfarnung https://github.com/nodejs/node-chakracore/pull/431 is the relevant PR

Reply:
@digitalinfinity this one is already embedded inside a `node_engine=="v8"` condition, so this should be unnecessary.  I think the bug you were seeing was caused by the unnecessary duplicate entry:

https://github.com/nodejs/node-chakracore/pull/446/files#diff-259e627be3fafed9e9c10a7fcb26879eL998